include/ocpn_plugin.h

```diff
@@ -80,28 +80,74 @@ class wxGLCanvas;
 //    Bitfield PlugIn Capabilites flag definition
 //
 //---------------------------------------------------------------------------------------------------------
+/** Receive callbacks to render custom overlay graphics on the chart.
+    Used for drawing additional navigation data, markers, or custom
+   visualizations. */
```

Comment:
I've added `OnToolbarToolCallback` for `WANTS_TOOLBAR_CALLBACK`. For others, I agree it would be useful to provide more details. Perhaps in a follow-up PR?